dev/sample-trees.js

```diff
@@ -0,0 +1,220 @@
+                                "spans": [
+                                  {
+                                    "start": 17,
+                                    "end": 28
+                                  }
+                                ],
+                                "link": "seqChild"
+                              },
+                              {
+                                "nodeType": "detail",
+                                "word": "odd",
+                                "spans": [
+                                  {
+                                    "start": 29,
+                                    "end": 32
+                                  }
+                                ],
+                                "link": "seqChild"
+                              }
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                "nodeType": "numericstring",
+                "word": "9",
+                "link": "obj"
+              }
+            ]
+          },
+          {
+            "nodeType": "event",
+            "word": "be",
+            "attributes": ["linking"],
+            "link": "none",
+            "spans": [
+              {
+                "start": 53,
+                "end": 55
+              }
+            ],
+            "children": [
+              {
+                "nodeType": "entity",
+                "word": "what",
+                "spans": [
+                  {
+                    "start": 48,
+                    "end": 52
+                  }
+                ],
+                "link": "subj",
+              },
+              {
+                "nodeType": "detail",
+                "word": "large",
+                "spans": [
+                  {
+                    "start": 56,
+                    "end": 67
+                  }
+                ],
+                "attributes": ["superlative"],
+                "link": "advx"
+              },
+              {
+                "nodeType": "detail",
+                "word": "Integer",
+                "spans": [
+                  {
+                    "start": 68,
+                    "end": 75
+                  }
+                ],
+                "attributes": [],
+                "link": "obj"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      text: 'Sam likes bananas',
+      root: {
+        nodeType: 'event',
+        word: 'like',
+        children: [
+          {
+            nodeType: 'entity',
+            word: 'Sam',
+            link: 'subject',
+            attributes: [ 'Person' ]
+          },
+          {
+            nodeType: 'entity',
+            word: 'banana',
+            link: 'object',
+            attributes: [ '>1']
+          }
+        ]
+      }
+    }
+  ];
+
```

Comment:
I see why you're doing this, but my concern with putting multiple trees in a single object is that it would become unruly for users who might want to play around with these examples. I could see us adding at least a handful more trees for demo purposes.

Reply:
Do you think there'd be any readability benefits to splitting each tree into its own `...tree.js` file?